static/js/components/tiles/answer_message_tile.tsx

```diff
@@ -40,43 +41,88 @@ export interface AnswerMessageTilePropType {
   displayValue?: DisplayValueSpec;
 }
 
+interface AnswerMessageTileData {
+  // Inline value to show as the answer
+  displayValue: DisplayValueSpec;
+  // Name of the entity to show the answer for
+  entityName: string;
+}
+
+/**
+ * Bold and italicize an entity's name in the title
+ * @param title tile title containing entity's name
+ * @param entityName name of the entity to highlight
+ * @returns JSX element containing stylized element
+ */
+function highlightEntities(title: string, entityName: string): JSX.Element {
+  const entityIndex = title.indexOf(entityName);
+  if (entityIndex == -1) {
+    return <span>{title}</span>;
+  }
+  return (
+    <span>
+      {title.slice(0, entityIndex)}
+      <b>
+        <i>{entityName}</i>
+      </b>
+      {title.slice(entityIndex + entityName.length, title.length)}
+    </span>
+  );
+}
+
 export function AnswerMessageTile(
   props: AnswerMessageTilePropType
 ): JSX.Element {
+  // Values to display
   const [answerData, setAnswerData] = useState<DisplayValueSpec | undefined>(
     props.displayValue
   );
+  // Name of entity the tile is about
+  const [entityName, setEntityName] = useState<string>("");
 
   useEffect(() => {
     fetchData(props).then((data) => {
-      setAnswerData(data);
+      setAnswerData(data.displayValue);
+      setEntityName(data.entityName);
     });
   }, [props]);
 
   if (!answerData) {
     return null;
   }
 
+  const answerValues = answerData.values.join(", ");
+
   return (
     <div
       className={`chart-container answer-message-tile ${ASYNC_ELEMENT_HOLDER_CLASS}`}
     >
       <div className={`answer-message ${ASYNC_ELEMENT_CLASS}`}>
-        <span>{props.title}</span>
-        <span>{answerData.values.join(", ")}</span>
+        {highlightEntities(props.title, entityName)}
+        <span>{answerValues}</span>
       </div>
-      <div className="source">source: {answerData.sources.join(", ")}</div>
+      <div className="source">Source: {answerData.sources.join(", ")}</div>
+      <CopyButton textToCopy={answerValues}></CopyButton>
     </div>
   );
 }
 
 const fetchData = async (
   props: AnswerMessageTilePropType
-): Promise<DisplayValueSpec> => {
-  if (props.displayValue || _.isEmpty(props.propertyExpr)) {
-    return Promise.resolve(props.displayValue);
+): Promise<AnswerMessageTileData> => {
+  let entityName = "";
+  if (_.isEmpty(props.propertyExpr)) {
```

Comment:
also do this if `props.displayValue` is true?

Reply:
Added check for props.displayValue